Storage/Storage/Model/DashboardCard.swift

```diff
@@ -6,12 +6,31 @@ public struct DashboardCard: Codable, Hashable, Equatable, GeneratedCopiable {
     /// The type of dashboard card.
     public let type: CardType
 
-    /// Whether the card is enabled in the Analytics Hub.
+    /// The card's availability state for the site.
+    /// To be set externally based on each card's availability check.
+    public var isAvailable: Bool
+
+    /// User-changeable setting in the Customize screen, whether to enable or disable an available card.
+    /// An available card will become invisible on the Dashboard, but stay visible on Customize, if `enabled` is set to false.
     public var enabled: Bool
```

Comment:
I missed this when writing this file but we should keep this immutable too.

Reply:
fixed in https://github.com/woocommerce/woocommerce-ios/pull/12626/commits/4d3466adef2a24904643d0b19c73dfdd118f2a78